Adafruit_SSD1306.h

```diff
@@ -24,7 +24,7 @@ All text above, and the splash screen must be included in any redistribution
   #define WIRE_WRITE Wire.send
 #endif
 
-#ifdef __SAM3X8E__
+#if !defined(__SAM3X8E__) &&  !defined(ENERGIA)
```

Comment:
That part of this PR seems wrong -- it changes the behavior on __SAM3X8E__ (it changes from "if __SAM3X8E__ defined" to "if __SAM3X8E__ NOT defined"